README.md

```diff
@@ -209,22 +210,26 @@ The service can be started against a set of non-existent database. If no pre-exi
 4. Start the Admin project and this will configure the contained users' permissions via database migrations. 
    The other projects will then be able to be started, using their own contained users to connect to the databases. 
 
+### Setting up the storage emulator
+The Azurite Docker container can be started by one of the following methods:
+1. Indirectly by starting the admin via the start script
+```bash
+    pnpm start admin
+```
+2. Directly via the start script using pnpm start dataStorage
```

Comment:
No need to repeat the command here. Can simplify this to:

2. Directly via the `start` script using: